common-libs/common/test/unit/utils/objectUtil.test.ts

```diff
@@ -0,0 +1,62 @@
+import { buildObjectSkeletonFromPaths, injectFieldForAllParentRoots } from '../../../src/utils/objectUtil'
+import { expect } from 'chai'
+
+describe('objectUtil', () => {
+  describe('#buildObjectSkeletonFromPaths', () => {
+    it('should build object that represents paths to fields', () => {
+      const paths = ['address/buildingNumber', 'address/country', 'id/age', 'id/name/firstName', 'date']
+
+      const representingPathsAsObject = buildObjectSkeletonFromPaths(paths)
+
+      expect(representingPathsAsObject).to.deep.equal({
+        date: {},
+        address: {
+          buildingNumber: {},
+          country: {},
```

Comment:
Is there any documentation reference for why `country` should be `{}` instead of e.g. `null`?

Reply:
To be honest, I haven't try to use not `{}` due to examples contains `{}`
e.g.: [example of revealDoc without nested fields](https://github.com/mattrglobal/jsonld-signatures-bbs/blob/master/sample/ts-node/src/data/deriveProofFrame.json#L11), [example of revealDoc with nested fields](https://github.com/mattrglobal/jsonld-signatures-bbs/blob/a1271750ed5211f1210323534402575967e4fae2/__tests__/__fixtures__/data/test_nested_reveal_document.json#L15)